spec/models/game_spec.rb

```diff
@@ -43,4 +43,40 @@
       expect(game.white_to_move).to eq(true)
     end
   end
+
+  describe "Populate Board" do
+    it "should have correct number of pieces" do
+      expect(game.pieces.count).to eq 32
+    end
+
+    it "should be empty on the squares" do
+      expect(game.pieces.where(rank: 3..6)).to eq([])
+    end
+
+    (1..8).each do |file|
+      it "should have a white pawn at rank 2, file #{file}" do
+        white_pawn = game.pieces.find{|piece| piece.rank == 2 && piece.file == file}
+        expected = ["Pawn", "white"]
+        expect([white_pawn.type, white_pawn.color]).to eq(expected)
+      end
+
+      it "should have a black pawn at rank 7, file #{file}" do
+        black_pawn = game.pieces.find{|piece| piece.rank == 7 && piece.file == file}
+        expected = ["Pawn", "black"]
+        expect([black_pawn.type, black_pawn.color]).to eq(expected)
+      end
+    end
+
+    {1 => "Rook", 2 => "Knight", 3 => "Bishop", 4 =>"Queen", 5 => "King", 6=> "Bishop", 7 => "Knight", 8 => "Rook"}.each do |file, type|  
+      it "should have a correct type of white piece at rank 1, file #{file}" do
+        white_piece = game.pieces.find{|piece| piece.rank == 1 && piece.file == file}
+        expect(white_piece.type).to eq(type)
+      end
```

Comment:
You should also check that the piece is the right color, and before you do that, I would check that the square isn't completely empty. This applies to all the examples that expect to find a piece.